.github/workflows/gpt4_summarizer.yml

```diff
@@ -0,0 +1,38 @@
+name: Summarize Supabase with GPT-4 and ZenML
+
+on:
+  schedule:
+    - cron: '0 */12 * * *'
+  push:
+    branches:
+      - project/new-summary-project
```

Comment:
You might want to keep the branch after merging so you can manually trigger it again then

Reply:
Yeah ill keep it around